rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ComponentSpecTest.java

```diff
@@ -946,6 +973,12 @@ private RestDocumentationResultHandler documentComponentProperties() {
                         fieldWithPath("ownerCountry").description("The owner country of the component"),
                         subsectionWithPath("externalIds").description("When projects are imported from other tools, the external ids can be stored here. Store as 'Single String' when single value, or 'Array of String' when multi-values"),
                         subsectionWithPath("additionalData").description("A place to store additional data used by external tools"),
+                        fieldWithPath("modifiedOn").description("The date the component was modified"),
+                        fieldWithPath("softwarePlatforms").description("The Software Platforms of componnet"),
+                        fieldWithPath("subscribers").description("The subscribers of componnet"),
+                        fieldWithPath("mainLicenseIds").description("The Main License Ids of componnet"),
+                        fieldWithPath("wiki").description("The wiki of componnet"),
+                        fieldWithPath("blog").description("The blog of componnet"),
```

Comment:
Please correct the component spelling in the above lines wherever component is used in description.